blueprint/client-devices.md

```diff
@@ -659,7 +659,7 @@ It is vital to have a robust and reliable patch management solution based on ind
 For Microsoft Windows environments the primary patching technologies are:
 
 * Windows Server Update Service – WSUS enables administrators to deploy the most recent Microsoft updates. A WSUS server connects directly to Microsoft Update or an "upstream" WSUS server. This allows administrators to control what updates are applied and when, rather than having every computer on the network going to the Internet and installing every available update immediately
-* Microsoft System Centre Configuration Manager (SCCM) –SCCM integrates with a WSUS server to deliver patch management. WSUS obtains updates from the internet and SCCM is used to approve and deploy the updates. Using SCCM to deploy software updates allows for more control over many aspects of the process such as targeting, maintenance windows, scheduling, and reporting
```

Comment:
I think the product name is in US spelling: `System Center`

Reply:
This issue is not in my code - it has been updated to Microsoft Endpoint Configuration Manager